composer.json

```diff
@@ -24,8 +24,8 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.6.2 || ^0.7 || ^1.0.0",
-        "slevomat/coding-standard": "^8.16",
-        "squizlabs/php_codesniffer": "^3.7"
+        "slevomat/coding-standard": "^8.23",
+        "squizlabs/php_codesniffer": "^4"
```

Comment:
Do we have make a hard bump here or could we support both major versions at the same time?

Reply:
E.g. this definition is not v3/v4 cross compatible.

<img width="1343" height="140" alt="image" src="https://github.com/user-attachments/assets/bf800fc2-c122-4b17-a1f4-1909642c8220" />

And also what @kukulich said.

Reply:
ok, that I missed. But still, this CS relies heavily on Slevomat's CS which requires v4 so allowing v3 here has no point if I'm not missing anything.